dockerfiles/run/guice/memory/destination/conf/deletedMessageVault.properties

```diff
@@ -1 +1,4 @@
-urlPrefix=memory://var/deletedMessages/vault
```

Comment:
why have you dropped it?

Reply:
Because we don't use it in the new implementation? We only use the BlobStore now, and the fact that if we use object storage or cassandra and its conf is configurated in blob.properties. Thus, `urlPrefix` became obsolete